packages/flame/lib/src/components/core/component.dart

```diff
@@ -375,8 +376,8 @@ class Component {
   ///   - it is invoked when the size of the game canvas is already known.
   ///
   /// If your loading logic requires knowing the size of the game canvas, then
-  /// add `HasGameRef` mixin and then query `gameRef.size` or
-  /// `gameRef.canvasSize`.
+  /// add [HasGameReference] mixin and then query `game.size` or
```

Comment:
This should still reference `HasGameRef` since the other one still is in experimental
```suggestion
  /// add [HasGameRef] mixin and then query `game.size` or
```